app-config.yaml

```diff
@@ -88,10 +88,6 @@ catalog:
     - type: url
       target: https://github.com/backstage/backstage/blob/master/catalog-info.yaml
 
-    # The demo site
-    - type: url
-      target: https://github.com/backstage/demo/blob/master/catalog-info.yaml
-
```

Comment:
This would remove the demo entity from the catalog. Is this intentional?

Reply:
Sorry for the slow reply, I removed it as it looked like the same things as line 85 above

Reply:
Oh, you're right, I missed that. :) Coo coo